src/ipc/prepare-boot-usb.ts

```diff
@@ -1,14 +1,14 @@
 import makeDebug from 'debug'
-import { IpcMain, IpcMainInvokeEvent } from 'electron'
+import { IpcMain } from 'electron'
 import exec from '../utils/exec'
-import { join } from 'path'
 
 export const channel = 'prepare-boot-usb'
 const debug = makeDebug('kiosk-browser:prepare-boot-usb')
 
-const BootableOptionPattern = /^Boot([0-9]+)\*\s(.+)\s(.+)$/
+const BootableOptionPattern = /^Boot([0-9a-fA-F]+)\*?\s(.+)\s(.+)$/
```

Comment:
I think the `\s` should be `\s+` in case there's more than one space. I've seen some weird entries.